### PR TITLE
Remove CBO elasticities toggle

### LINKED ISSUE #5310: Remove CBO elasticities toggle
We should instead be using the reform for greater transparency: https://policyengine.org/us/policy?focus=gov&reform=62273&region=us&timePeriod=2024&baseline=2

### PR DESCRIPTION
**Pull request by vrathi101**
Fixes #5310

### CODE REVIEW

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_us/variables/gov/simulation/labor_supply_response/labor_supply_response.py [hunk 1]
```diff
@@ -95,15 +95,11 @@ class income_elasticity(Variable):
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/simulation/labor_supply_response/labor_supply_response.py:95]: Also change unit to "/1" (substitution too)

```suggestion
        adds = ["gov.simulation.labor_supply_responses.elasticities.income"]
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
Could you also add unit tests while here to appease the codecov?

**Comment by MaxGhenis** [comment]
Friendly ping

**Comment by vrathi101** [comment]
> Friendly ping

Me and @PavelMakarchuk were working through how to write the unit tests for `labor_supply_response.py` because of some input issues with the class structure

**Comment by PavelMakarchuk** [comment]
> > Friendly ping
> 
> Me and @PavelMakarchuk were working through issues with how to write the unit tests for `labor_supply_response.py` because of some input issues with the class structure

@MaxGhenis Yes I would file a separate issue to add testing for labor supply related variables as we can't seem to accommodate the `requires_computation_after` metadata

**Review by MaxGhenis — APPROVED** [review]

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
please fix the unit of the elasticities per my previous comment

**Comment by MaxGhenis** [comment]
Responses should stay USD, elasticities are /1

**Pull request merged**